### DISCUSSION TITLE
Welcome to bypy Discussions!

### DISCUSSION THREAD

**Discussion by houtianze** [houtianze/bypy#561] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by nyliubing** [comment]
![2021-08-18_12-51](https://user-images.githubusercontent.com/85790815/129838690-a59525ec-cf07-4a85-ab3f-46ae21b43367.png)
经常出现这种情况

**Reply by liangwx** [reply]
我也遇到了，请问怎么解决

**Comment by Vimos** [comment]
作为超级会员，我发现上传速度并不稳定，有时候只在500kb以下。
`[===========_________] 55% (600.0MB/1.1GB) ETA: 17m8s (476kB/s, 21m30s gone)  
`
不知道是不是使用的时候少了什么配置

**Reply by liangwx** [reply]
请问路过的大佬们，超级会员结合aria2的稳定下载与上传速度可以达到多少？

**Comment by liu-qingyuan** [comment]
下载速度怎么解决啊，在colab上下个1G的数据集要10多分钟，救命!!
![QQ截图20221009180000](https://user-images.githubusercontent.com/57737268/194750465-199a691f-b16f-436c-ab7f-f6ad0549f52c.png)

**Reply by liu-qingyuan** [reply]
我也是超级会员

**Reply by mmcx-zsy** [reply]
加上downloader 参数
bypy --downloader aria2 download filename
需要自行安装 aria2

**Reply by liangwx** [reply]
请问路过的大佬们，超级会员结合aria2的稳定下载与上传速度可以达到多少？

**Reply by Zxl19990529** [reply]
@liangwx  我也是超级会员，不用aria2是4mb/s，用了是20mb/s。
